Dz3/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection.Metadata.Ecma335;
+using System.Text.RegularExpressions;
+
+namespace DZ2DZ3
+{
+    public class Program
+    {
+        public enum Gender
+        {
+            male,
+            female,
+            undefined
+        }
+        static void Main(string[] args)
+        {
+            User user1 = new User();
+            // user1.Gender = Gender.Male;
+            try
+            {
+                while (true)
+                {
+                    Console.WriteLine(" Pleace input your name");
+                    user1.FirstName = Console.ReadLine();
+
+                    if (Regex.IsMatch(user1.FirstName, "^[a-zA-Z]+$"))
+                    {
+                        user1.FirstName = user1.FirstName;
+                        break;
+                    }
+                    else
+                    {
+                        Console.WriteLine("Invalid first name! Please enter only letters.");
+                    }
+                }
+                    Console.WriteLine(" Pleace input your surname");
+                    user1.LastName = Console.ReadLine();
+                
+            }
+            catch
+            {
+                Console.WriteLine("You entered unkorect value.");
+            }
+          user1.Name();
```

Comment:
And what is that? In the task said: override ToString() method. — It is not an override of ToString() method... It is something different... Very different.

Reply:
we don`t have any task to due about override in our homework, so...